# Hungarian translation for backintime
# Copyright (c) 2009 Rosetta Contributors and Canonical Ltd 2009
# This file is distributed under the same license as the backintime package.
#
# FIRST AUTHOR <EMAIL@ADDRESS>, 2009.
# Kósa Lajos <lajos@kosa.co.hu>, 2013.
msgid ""
msgstr ""
"Project-Id-Version: backintime\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2016-02-12 03:33+0100\n"
"PO-Revision-Date: 2016-02-12 04:25+0000\n"
"Last-Translator: Germar <Unknown>\n"
"Language-Team: Hungarian <kde-l10n-hu@kde.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2016-02-12 04:32+0000\n"
"X-Generator: Launchpad (build 17923)\n"
"Language: hu\n"

#: ../../common/bindfstools.py:62 ../../common/sshtools.py:114
#, python-format
msgid "Can't mount %s"
msgstr "Nem tudom felcsatolni: %s"

#: ../../common/configfile.py:97
#, python-format
msgid "Failed to save config: %s"
msgstr ""

#: ../../common/configfile.py:128
#, python-format
msgid "Failed to load config: %s"
msgstr ""

#: ../../common/configfile.py:601 ../../common/configfile.py:684
#, python-format
msgid "Profile \"%s\" already exists !"
msgstr "A \"%s\" már létezik!"

#: ../../common/configfile.py:641
msgid "You can't remove the last profile !"
msgstr "Nem távolíthatod el az utolsó profilt!"

#: ../../common/config.py:77
msgid "Disabled"
msgstr "Kikapcsolva"

#: ../../common/config.py:78
msgid "At every boot/reboot"
msgstr "Minden (újra)indításkor"

#: ../../common/config.py:79
msgid "Every 5 minutes"
msgstr "Ötpercenként"

#: ../../common/config.py:80
msgid "Every 10 minutes"
msgstr "Tízpercenként"

#: ../../common/config.py:81
msgid "Every 30 minutes"
msgstr "30 percenként"

#: ../../common/config.py:82
msgid "Every hour"
msgstr "Óránként"

#: ../../common/config.py:83
msgid "Every 2 hours"
msgstr "Kétóránként"

#: ../../common/config.py:84
msgid "Every 4 hours"
msgstr "Négyóránként"

#: ../../common/config.py:85
msgid "Every 6 hours"
msgstr "Hatóránként"

#: ../../common/config.py:86
msgid "Every 12 hours"
msgstr "Tizenkétóránként"

#: ../../common/config.py:87
msgid "Custom Hours"
msgstr "Egyedi beállítás (óra)"

#: ../../common/config.py:88
msgid "Every Day"
msgstr "Naponta"

#: ../../common/config.py:89
msgid "Repeatedly (anacron)"
msgstr ""

#: ../../common/config.py:90
msgid "When drive get connected (udev)"
msgstr "Ha csatlakozik a meghajtó (udev)"

#: ../../common/config.py:91
msgid "Every Week"
msgstr "Hetente"

#: ../../common/config.py:92
msgid "Every Month"
msgstr "Havonta"

#: ../../common/config.py:96 ../../common/config.py:103
msgid "Day(s)"
msgstr "Nap"

#: ../../common/config.py:97 ../../common/config.py:104
msgid "Week(s)"
msgstr "Hét"

#: ../../common/config.py:98
msgid "Year(s)"
msgstr "Év"

#: ../../common/config.py:102
msgid "Hour(s)"
msgstr ""

#: ../../common/config.py:105
msgid "Month(s)"
msgstr ""

#: ../../common/config.py:127 ../../qt4/settingsdialog.py:552
msgid " EXPERIMENTAL!"
msgstr " KÍSÉRLETI!"

#: ../../common/config.py:130
msgid "Local"
msgstr "Helyi"

#: ../../common/config.py:131
msgid "SSH"
msgstr "SSH"

#: ../../common/config.py:131 ../../common/config.py:133
#: ../../qt4/settingsdialog.py:1589
msgid "SSH private key"
msgstr "SSH privát kulcs"

#: ../../common/config.py:132
msgid "Local encrypted"
msgstr "Helyi titkosított"

#: ../../common/config.py:132 ../../common/config.py:133
msgid "Encryption"
msgstr "Titkosítás"

#: ../../common/config.py:133
msgid "SSH encrypted"
msgstr "SSH-val titkosított"

#: ../../common/config.py:136
msgid "Default"
msgstr "Alapértelmezett"

#: ../../common/config.py:137
msgid "AES128-CTR"
msgstr "AES128-CTR"

#: ../../common/config.py:138
msgid "AES192-CTR"
msgstr "AES192-CTR"

#: ../../common/config.py:139
msgid "AES256-CTR"
msgstr "AES256-CTR"

#: ../../common/config.py:140
msgid "ARCFOUR256"
msgstr "ARCFOUR256"

#: ../../common/config.py:141
msgid "ARCFOUR128"
msgstr "ARCFOUR128"

#: ../../common/config.py:142
msgid "AES128-CBC"
msgstr "AES128-CBC"

#: ../../common/config.py:143
msgid "3DES-CBC"
msgstr "3DES-CBC"

#: ../../common/config.py:144
msgid "Blowfish-CBC"
msgstr "Blowfish-CBC"

#: ../../common/config.py:145
msgid "Cast128-CBC"
msgstr "Cast128-CBC"

#: ../../common/config.py:146
msgid "AES192-CBC"
msgstr "AES192-CBC"

#: ../../common/config.py:147
msgid "AES256-CBC"
msgstr "AES256-CBC"

#: ../../common/config.py:148
msgid "ARCFOUR"
msgstr "ARCFOUR"

#: ../../common/config.py:154
msgid "Main profile"
msgstr "Fő profil"

#: ../../common/config.py:249 ../../common/config.py:256
#: ../../common/config.py:267 ../../common/config.py:272
#: ../../qt4/qt4systrayicon.py:61
#, python-format
msgid "Profile: \"%s\""
msgstr "Profil: \"%s\""

#: ../../common/config.py:249
msgid "Snapshots folder is not valid !"
msgstr "A pillanatkép-könyvtár nem érvényes"

#: ../../common/config.py:256
msgid "You must select at least one folder to backup !"
msgstr "Legalább egy könyvtárat ki kell jelölnie a mentéshez"

#: ../../common/config.py:267
msgid "You can't include backup folder !"
msgstr "Nem adható hozzá a biztonsági mentés mappája"

#: ../../common/config.py:272
msgid "You can't include a backup sub-folder !"
msgstr "Nem adható hozzá a biztonsági mentés almappája"

#: ../../common/config.py:330
#, python-format
msgid "%s is not a folder !"
msgstr "A(z) %s nem könyvtár"

#: ../../common/config.py:339
msgid "Host/User/Profile-ID must not be empty!"
msgstr ""

#: ../../common/config.py:347 ../../common/config.py:379
#, python-format
msgid ""
"Can't write to: %s\n"
"Are you sure you have write access ?"
msgstr ""
"Nem tudok ide írni: %s\n"
"Biztos, hogy van írási jogod?"

#: ../../common/config.py:359
#, python-format
msgid ""
"Destination filesystem for '%(path)s' is formatted with FAT which doesn't "
"support hard-links. Please use a native Linux filesystem."
msgstr ""

#: ../../common/config.py:364
#, python-format
msgid ""
"Destination filsystem for '%(path)s' is a SMB mounted share. Please make "
"sure the remote SMB server supports symlinks or activate '%(copyLinks)s' in "
"'%(expertOptions)s'."
msgstr ""

#: ../../common/config.py:367 ../../qt4/settingsdialog.py:816
msgid "Copy links (dereference symbolic links)"
msgstr "Linkek másolása (szimbolikus linkeknél nem csak a hivatkozást)"

#: ../../common/config.py:368 ../../qt4/settingsdialog.py:670
msgid "Expert Options"
msgstr "Szakértői beállítások"

#: ../../common/config.py:370
#, python-format
msgid ""
"Destination filesystem for '%(path)s is a sshfs mounted share. sshfs doesn't "
"support hard-links. Please use mode 'SSH' instead."
msgstr ""

#: ../../common/config.py:1378
msgid ""
"Can't find crontab.\n"
"Are you sure cron is installed ?\n"
"If not you should disable all automatic backups."
msgstr ""
"A crontab nem található.\n"
"Biztos, hogy telepítve van a cron?\n"
"Ha nem, akkor ki kell kapcsolni az automatikus biztonsági mentést."

#: ../../common/config.py:1382
msgid "Failed to write new crontab."
msgstr ""

#: ../../common/config.py:1476
#, python-format
msgid ""
"Could not install Udev rule for profile %(profile_id)s. DBus Service "
"'%(dbus_interface)s' wasn't available"
msgstr ""

#: ../../common/config.py:1488
#, python-format
msgid "Schedule udev doesn't work with mode %s"
msgstr "Az udev időzítése nem működik ezzel a móddal: %s"

#: ../../common/config.py:1497
#, python-format
msgid "Couldn't find UUID for \"%s\""
msgstr "Nem találom a \"%s\" meghajtó UUID-ját"

#: ../../common/encfstools.py:89
#, python-format
msgid ""
"Can't mount '%(command)s':\n"
"\n"
"%(error)s"
msgstr ""
"Nem tudtam felcsatolni a következőt: '%(command)s':\n"
"\n"
"%(error)s"

#: ../../common/encfstools.py:137
msgid "Config for encrypted folder not found."
msgstr "A titkosított mappa konfigurációja nem található"

#: ../../common/encfstools.py:141
msgid ""
"\n"
"Create a new encrypted folder?"
msgstr ""
"\n"
"Létrehozzak egy új titkosított mappát?"

#: ../../common/encfstools.py:142
msgid "Cancel"
msgstr "Mégsem"

#: ../../common/encfstools.py:145
msgid "Please confirm password"
msgstr "Erősítsd meg a jelszót"

#: ../../common/encfstools.py:149
msgid "Password doesn't match"
msgstr "A jelszavakat nem egyeznek"

#: ../../common/encfstools.py:166
msgid ""
"encfs version 1.7.2 and before has a bug with option --reverse. Please "
"update encfs"
msgstr ""
"Az encfs az 1.7.2-es és korábbi verzióknál hibás, ha a --reverse opciót "
"használod. Frissítsd az encfs-t"

#: ../../common/encfstools.py:498 ../../common/snapshots.py:875
#: ../../qt4/app.py:79
msgid "Take snapshot"
msgstr "Pillanatkép készítése"

#: ../../common/mount.py:431
#, python-format
msgid ""
"Hash collision occurred in hash_id %s. Incrementing global value "
"hash_collision and try again."
msgstr ""
"Hash-ütközés történt: hash_id %s. Megnövelem a hash_collision értékét, és "
"újra próbálom."

#: ../../common/mount.py:506
#, python-format
msgid "Can't unmount %(proc)s from %(mountpoint)s"
msgstr ""

#: ../../common/mount.py:593
#, python-format
msgid "%(proc)s not found. Please install e.g. %(install_command)s"
msgstr ""

#: ../../common/mount.py:606
#, python-format
msgid ""
"%(user)s is not member of group 'fuse'.\n"
" Run 'sudo adduser %(user)s fuse'. To apply changes logout and login again.\n"
"Look at 'man backintime' for further instructions."
msgstr ""
"A(z) %(user)s nem tagja a 'fuse' csoportnak.\n"
" Futtasd le a 'sudo adduser %(user)s fuse' parancsot, majd jelentkezz ki és "
"vissza.\n"
"Nézd meg a 'man backintime'-ot további részletekért."

#: ../../common/mount.py:627
#, python-format
msgid "mountpoint %s not empty."
msgstr "A %s csatolási pont foglalt"

#: ../../common/mount.py:690
msgid "Mountprocess lock timeout"
msgstr "A csatolási folyamat túl sok időt vett igénybe"

#: ../../common/password.py:387
#, python-format
msgid "Profile '%(profile)s': Enter password for %(mode)s: "
msgstr ""
"A(z) '%(profile)s' profil: add meg a jelszót a következőhöz: %(mode)s "

#: ../../common/snapshots.py:156
msgid ""
"### This log has been decoded with automatic search pattern\n"
"### If some paths are not decoded you can manually decode them with:\n"
msgstr ""
"### Ezt a naplófájlt automatikus keresési mintával dekódolták\n"
"### Ha néhány útvonal dekódolása elmaradt, a következővel tudod kézzel "
"dekódolni:\n"

#: ../../common/snapshots.py:393 ../../common/snapshots.py:629
msgid "FAILED"
msgstr "MEGHIÚSULT"

#: ../../common/snapshots.py:576 ../../common/snapshots.py:632
msgid "Restore permissions:"
msgstr "Engedélyek visszaállítása:"

#: ../../common/snapshots.py:631 ../../qt4/app.py:380 ../../qt4/app.py:617
#: ../../qt4/app.py:636 ../../qt4/qt4systrayicon.py:65
msgid "Done"
msgstr "Kész"

#: ../../common/snapshots.py:699
msgid "Deferring backup while on battery"
msgstr ""

#: ../../common/snapshots.py:756
msgid ""
"Can't find snapshots folder.\n"
"If it is on a removable drive please plug it."
msgstr ""
"A pillanatképek könyvtára nem található.\n"
"Ha egy eltávolítható adathordozón van, csatlakoztasd."

#: ../../common/snapshots.py:758
#, python-format
msgid "Waiting %s second."
msgid_plural "Waiting %s seconds."
msgstr[0] "%s másodperc várakozás."
msgstr[1] "%s másodperc várakozás."

#: ../../common/snapshots.py:783
#, python-format
msgid "Failed to take snapshot %s !!!"
msgstr "A %s pillanatkép készítése meghiúsult!!!"

#: ../../common/snapshots.py:792
msgid "Finalizing"
msgstr "Véglegesítés"

#: ../../common/snapshots.py:923
#, python-format
msgid "Can't create folder: %s"
msgstr "A(z) %s mappa nem hozható létre"

#: ../../common/snapshots.py:971
msgid "..."
msgstr "..."

#: ../../common/snapshots.py:979
#, python-format
msgid "Found leftover '%s' which can be continued."
msgstr ""

#: ../../common/snapshots.py:990
#, python-format
msgid "Removing leftover '%s' folder from last run"
msgstr ""

#: ../../common/snapshots.py:1003
#, python-format
msgid "Can't remove folder: %s"
msgstr "A(z) %s mappa nem távolítható el"

#: ../../common/snapshots.py:1033
#, python-format
msgid "Comparing with snapshot %s"
msgstr "Összehasonlítás a következő pillanatképpel: %s"

#: ../../common/snapshots.py:1053
msgid "Create hard-links"
msgstr "Hard-linkek készítése"

#: ../../common/snapshots.py:1087
msgid "Taking snapshot"
msgstr "Pillanatkép készítése"

#: ../../common/snapshots.py:1145
msgid "Saving config file..."
msgstr "Konfigurációs fájl mentése..."

#: ../../common/snapshots.py:1151
msgid "Saving permissions..."
msgstr "Hozzáférési jogosultságok mentése..."

#: ../../common/snapshots.py:1175
msgid "Save permission over ssh failed. Retry normal method"
msgstr ""

#: ../../common/snapshots.py:1225
#, python-format
msgid "Can't rename %(new_path)s to %(path)s"
msgstr "Nem tudom a(z) %(path)s útvonalat átnevezni erre: %(new_path)s"

#: ../../common/snapshots.py:1483 ../../common/snapshots.py:1529
#: ../../qt4/settingsdialog.py:543
msgid "Smart remove"
msgstr "Okos eltávolítás"

#: ../../common/snapshots.py:1506
msgid "Removing old snapshots"
msgstr "Régi pillanatképek eltávolítása"

#: ../../common/snapshots.py:1534
msgid "Trying to keep min free space"
msgstr "A minimálisan megadott szabad hely kialakítása"

#: ../../common/snapshots.py:1571
#, python-format
msgid "Trying to keep min %d%% free inodes"
msgstr "Megpróbálok legalább %d%% szabad inode-ot tartani"

#: ../../common/snapshots.py:2235
msgid "WITH ERRORS !"
msgstr "HIBÁKKAL!"

#: ../../common/snapshots.py:2669 ../../qt4/app.py:206 ../../qt4/app.py:1149
msgid "Now"
msgstr "Most"

#: ../../common/sshtools.py:208
msgid ""
"Could not unlock ssh private key. Wrong password or password not available "
"for cron."
msgstr ""
"Nem tudtam feloldani az ssh privát kulcsát. Hibás jelszó vagy a cron nem éri "
"el a jelszót."

#: ../../common/sshtools.py:226
#, python-format
msgid ""
"Password-less authentication for %(user)s@%(host)s failed. Look at 'man "
"backintime' for further instructions."
msgstr ""
"A %(user)s@%(host)s nem tudott jelszó nélkül bejelentkeztni. Nézd meg a 'man "
"backintime'-ot további részletekért."

#: ../../common/sshtools.py:248
#, python-format
msgid ""
"Cipher %(cipher)s failed for %(host)s:\n"
"%(err)s"
msgstr ""
"A(z) %(cipher)s nem sikerült a %(host)s esetében:\n"
"%(err)s"

#: ../../common/sshtools.py:285
#, python-format
msgid "%s not found in ssh_known_hosts."
msgstr "A(z) %s nem található itt: ssh_known_hosts."

#: ../../common/sshtools.py:315
#, python-format
msgid ""
"Remote path exists but is not a directory:\n"
" %s"
msgstr ""
"A távoli útvonal létezik, de nem könyvtár:\n"
" %s"

#: ../../common/sshtools.py:317
#, python-format
msgid ""
"Remote path is not writeable:\n"
" %s"
msgstr ""
"A távoli útvonal nem írható:\n"
" %s"

#: ../../common/sshtools.py:319
#, python-format
msgid ""
"Remote path is not executable:\n"
" %s"
msgstr ""
"A távoli útvonal nem futtatható:\n"
" %s"

#: ../../common/sshtools.py:321
#, python-format
msgid ""
"Couldn't create remote path:\n"
" %s"
msgstr ""
"Nem tudom távoli útvonalat elkészíteni:\n"
" %s"

#: ../../common/sshtools.py:346
#, python-format
msgid "Ping %s failed. Host is down or wrong address."
msgstr ""
"A(z) %s ping sikertelen volt. A kiszolgáló nem működik vagy rossz a cím."

#: ../../common/sshtools.py:380 ../../common/sshtools.py:509
#, python-format
msgid ""
"Remote host %(host)s doesn't support '%(command)s':\n"
"%(err)s\n"
"Look at 'man backintime' for further instructions"
msgstr ""
"A(z) %(host)s távoli kiszolgáló nem támogatja a következőt: '%(command)s':\n"
"%(err)s\n"
"Nézd meg a 'man backintime'-ot további részletekért"

#: ../../common/sshtools.py:512
#, python-format
msgid ""
"Check commands on host %(host)s returned unknown error:\n"
"%(err)s\n"
"Look at 'man backintime' for further instructions"
msgstr ""
"A parancsok ellenőzése a(z) %(host)s kiszolgálón ismeretlen hibát okozott:\n"
"%(err)s\n"
"Nézd meg a 'man backintime'-ot további részletekért"

#: ../../common/sshtools.py:527
#, python-format
msgid "Remote host %s doesn't support hardlinks"
msgstr "A(z) %s kiszolgáló nem támogatja a hardlinkeket."

#: ../../qt4/app.py:82
msgid "Refresh snapshots list"
msgstr "Pillanatképek listájának frissítése"

#: ../../qt4/app.py:86 ../../qt4/app.py:814
msgid "Snapshot Name"
msgstr "Pillanatkép neve"

#: ../../qt4/app.py:89
msgid "Remove Snapshot"
msgstr "Pillanatkép eltávolítása"

#: ../../qt4/app.py:92
msgid "View Snapshot Log"
msgstr "A pillanatképek naplófájljának megtekintése"

#: ../../qt4/app.py:95
msgid "View Last Log"
msgstr "Utolsó naplófájl megtekintése"

#: ../../qt4/app.py:100 ../../qt4/settingsdialog.py:52
msgid "Settings"
msgstr "Beállítások"

#: ../../qt4/app.py:105
msgid "Shutdown"
msgstr "Leállítás"

#: ../../qt4/app.py:106
msgid "Shutdown system after snapshot has finished."
msgstr "Állítsd le a rendszert, ha elkészült a pillanatkép."

#: ../../qt4/app.py:112
msgid "Exit"
msgstr "Kilépés"

#: ../../qt4/app.py:121 ../../qt4/app.py:140 ../../qt4/app.py:246
msgid "Help"
msgstr "Súgó"

#: ../../qt4/app.py:123
msgid "Config File Help"
msgstr ""

#: ../../qt4/app.py:126
msgid "Website"
msgstr "Honlap"

#: ../../qt4/app.py:128 ../../qt4/app.py:905
msgid "Changelog"
msgstr ""

#: ../../qt4/app.py:130
msgid "FAQ"
msgstr "GYIK"

#: ../../qt4/app.py:132
msgid "Ask a question"
msgstr "Kérdezz"

#: ../../qt4/app.py:134
msgid "Report a bug"
msgstr "Hiba jelentése"

#: ../../qt4/app.py:137 ../../qt4/app.py:1268
msgid "About"
msgstr "Névjegy"

#: ../../qt4/app.py:164
msgid "Up"
msgstr "Fel"

#: ../../qt4/app.py:175 ../../qt4/settingsdialog.py:1783
msgid "Show hidden files"
msgstr "Rejtett fájlok megjelenítése"

#: ../../qt4/app.py:187 ../../qt4/app.py:202 ../../qt4/app.py:236
#: ../../qt4/restoredialog.py:62 ../../qt4/settingsdialog.py:1825
#: ../../qt4/snapshotsdialog.py:134 ../../qt4/snapshotsdialog.py:139
msgid "Restore"
msgstr "Visszaállítás"

#: ../../qt4/app.py:189 ../../qt4/restoredialog.py:35
#: ../../qt4/snapshotsdialog.py:136
msgid "Restore to ..."
msgstr "Visszaállítás ide..."

#: ../../qt4/app.py:197
msgid "Restore and delete new files"
msgstr ""

#: ../../qt4/app.py:204
#, python-format
msgid ""
"Restore selected file or folder.\n"
"If this button is grayed out this is most likely because \"%(now)s\" is "
"selected in left hand snapshots list."
msgstr ""

#: ../../qt4/app.py:209 ../../qt4/logviewdialog.py:68
#: ../../qt4/qt4tools.py:170 ../../qt4/snapshotsdialog.py:96
msgid "Snapshots"
msgstr "Pillanatképek"

#: ../../qt4/app.py:216
msgid "Snapshot"
msgstr ""

#: ../../qt4/app.py:227
msgid "View"
msgstr ""

#: ../../qt4/app.py:280
msgid "Shortcuts"
msgstr "Billentyűparancsok"

#: ../../qt4/app.py:298
msgid ""
"This folder doesn't exist\n"
"in the current selected snapshot!"
msgstr ""

#: ../../qt4/app.py:362
msgid "Add to Include"
msgstr ""

#: ../../qt4/app.py:363
msgid "Add to Exclude"
msgstr ""

#: ../../qt4/app.py:419
#, python-format
msgid ""
"%(appName)s is not configured. Would you like to restore a previous "
"configuration?"
msgstr ""

#: ../../qt4/app.py:440
msgid ""
"Can't find snapshots folder.\n"
"If it is on a removable drive please plug it and then press OK"
msgstr ""
"Nem található a pillanatképek könyvtára.\n"
"Ha egy eltávolítható lemezen van, csatlakoztasd és nyomd meg az OK-t"

#: ../../qt4/app.py:473
msgid ""
"If you close this window Back In Time will not be able to shutdown your "
"system when the snapshot has finished.\n"
"Do you really want to close?"
msgstr ""
"Ha bezárod ezt az ablakot, a Back In Time nem tudja lekapcsolni a rendszert "
"a biztonsági mentés elvégzése után.\n"
"Biztos bezárod?"

#: ../../qt4/app.py:609 ../../qt4/app.py:647
msgid "Working:"
msgstr "Dolgozik:"

#: ../../qt4/app.py:639
msgid "Done, no backup needed"
msgstr "Kész, nincs szükség biztonsági mentés készítésére"

#: ../../qt4/app.py:651
msgid "Error:"
msgstr "Hiba:"

#: ../../qt4/app.py:670 ../../qt4/qt4systrayicon.py:153
msgid "Sent:"
msgstr ""

#: ../../qt4/app.py:671 ../../qt4/qt4systrayicon.py:154
msgid "Speed:"
msgstr ""

#: ../../qt4/app.py:672 ../../qt4/qt4systrayicon.py:155
msgid "ETA:"
msgstr ""

#: ../../qt4/app.py:721
msgid "Global"
msgstr "Globális"

#: ../../qt4/app.py:722
msgid "Root"
msgstr "Root"

#: ../../qt4/app.py:723
msgid "Home"
msgstr "Home"

#: ../../qt4/app.py:738
msgid "Backup folders"
msgstr "Biztonsági mentés mappái"

#: ../../qt4/app.py:852
#, python-format
msgid ""
"Are you sure you want to remove the snapshot:\n"
"%s"
msgstr ""
"Biztos, hogy el akar távolítani a következő pillanatképet:\n"
"%s?"

#: ../../qt4/app.py:937
#, python-format
msgid ""
"Backup local files before overwriting or\n"
"removing with trailing '%(suffix)s'."
msgstr ""

#: ../../qt4/app.py:943
msgid ""
"Are you sure you want to remove all newer files in your original folder?"
msgstr ""

#: ../../qt4/app.py:947
msgid ""
"WARNING: deleting files in filesystem root could break your whole system!!!"
msgstr ""

#: ../../qt4/app.py:949
msgid "Files to be restored:"
msgstr ""

#: ../../qt4/app.py:964
msgid "Do you really want to restore this files(s):"
msgstr ""

#: ../../qt4/app.py:1150
msgid "View the current disk content"
msgstr "A lemez jelenlegi tartalmának megtekintése"

#: ../../qt4/app.py:1153
#, python-format
msgid "Snapshot: %s"
msgstr "Pillanatkép: %s"

#: ../../qt4/app.py:1154
#, python-format
msgid "View the snapshot made at %s"
msgstr "A %s időpontban észült pillanatkép megtekintése"

#: ../../qt4/app.py:1192
#, python-format
msgid "Restore '%s'"
msgstr "A(z) '%s' visszaállítása"

#: ../../qt4/app.py:1193
#, python-format
msgid "Restore '%s' to ..."
msgstr "A(z) '%s' visszaállítása ide..."

#: ../../qt4/app.py:1194
#, python-format
msgid "Restore '%s' and delete new files"
msgstr ""

#: ../../qt4/app.py:1293 ../../qt4/app.py:1310
msgid "Authors"
msgstr ""

#: ../../qt4/app.py:1294 ../../qt4/app.py:1313
msgid "Translations"
msgstr ""

#: ../../qt4/app.py:1295 ../../qt4/app.py:1316
msgid "License"
msgstr ""

#: ../../qt4/logviewdialog.py:50
msgid "Last Log View"
msgstr ""

#: ../../qt4/logviewdialog.py:52
msgid "Snapshot Log View"
msgstr ""

#: ../../qt4/logviewdialog.py:60 ../../qt4/settingsdialog.py:60
#: ../../qt4/settingsdialog.py:275 ../../qt4/settingsdialog.py:1912
msgid "Profile:"
msgstr "Profil:"

#: ../../qt4/logviewdialog.py:82
msgid "Filter:"
msgstr "Szűrő:"

#: ../../qt4/logviewdialog.py:88 ../../qt4/settingsdialog.py:660
msgid "All"
msgstr "Mindegyik"

#: ../../qt4/logviewdialog.py:89 ../../qt4/logviewdialog.py:91
#: ../../qt4/settingsdialog.py:658
msgid "Errors"
msgstr "Hibák"

#: ../../qt4/logviewdialog.py:89 ../../qt4/logviewdialog.py:92
msgid "Changes"
msgstr "Változások"

#: ../../qt4/logviewdialog.py:93
msgid "Informations"
msgstr "Információk"

#: ../../qt4/logviewdialog.py:102
msgid "[E] Error, [I] Information, [C] Change"
msgstr "[E] hiba, [I] információ, [C] változtat"

#: ../../qt4/logviewdialog.py:105
msgid "decode paths"
msgstr "dekódolás útvonalai"

#: ../../qt4/messagebox.py:56
msgid "Error"
msgstr ""

#: ../../qt4/messagebox.py:62 ../../qt4/messagebox.py:68
msgid "Question"
msgstr ""

#: ../../qt4/qt4systrayicon.py:69
msgid "Start BackInTime"
msgstr ""

#: ../../qt4/qt4systrayicon.py:124
msgid "Working..."
msgstr "Dolgozom..."

#: ../../qt4/qt4tools.py:190
msgid "Today"
msgstr "Ma"

#: ../../qt4/qt4tools.py:194
msgid "Yesterday"
msgstr "Tegnap"

#: ../../qt4/qt4tools.py:199
msgid "This week"
msgstr "Ezen a héten"

#: ../../qt4/qt4tools.py:203
msgid "Last week"
msgstr "Múlt héten"

#: ../../qt4/qt4tools.py:319
msgid "This is NOT a snapshot but a live view of your local files"
msgstr ""

#: ../../qt4/qt4tools.py:321
#, python-format
msgid "Last check %s"
msgstr ""

#: ../../qt4/restoredialog.py:75
msgid "Show full Log"
msgstr ""

#: ../../qt4/settingsdialog.py:69
msgid "Edit"
msgstr "Szerkesztés"

#: ../../qt4/settingsdialog.py:74
msgid "Modify for Full System Backup"
msgstr ""

#: ../../qt4/settingsdialog.py:78 ../../qt4/settingsdialog.py:452
msgid "Add"
msgstr ""

#: ../../qt4/settingsdialog.py:82 ../../qt4/settingsdialog.py:409
#: ../../qt4/settingsdialog.py:468
msgid "Remove"
msgstr ""

#: ../../qt4/settingsdialog.py:97
msgid "General"
msgstr "Általános"

#: ../../qt4/settingsdialog.py:107 ../../qt4/settingsdialog.py:1914
msgid "Mode:"
msgstr "Mód:"

#: ../../qt4/settingsdialog.py:120
#, python-format
msgid ""
"<b>Warning:</b> %(app)s uses EncFS for encryption. A recent security audit "
"revealed several possible attack vectors for this. Please take a look at 'A "
"NOTE ON SECURITY' in 'man backintime'."
msgstr ""

#: ../../qt4/settingsdialog.py:130 ../../qt4/settingsdialog.py:1574
msgid "Where to save snapshots"
msgstr "Hova mentse a pillanatképeket"

#: ../../qt4/settingsdialog.py:146
msgid "Folder"
msgstr ""

#: ../../qt4/settingsdialog.py:154
msgid "SSH Settings"
msgstr "Az SSH beállításai"

#: ../../qt4/settingsdialog.py:166 ../../qt4/settingsdialog.py:263
msgid "Host:"
msgstr "Kiszolgáló:"

#: ../../qt4/settingsdialog.py:171
msgid "Port:"
msgstr "Port:"

#: ../../qt4/settingsdialog.py:176 ../../qt4/settingsdialog.py:269
msgid "User:"
msgstr "Felhasználó:"

#: ../../qt4/settingsdialog.py:181
msgid "Path:"
msgstr "Elérési út:"

#: ../../qt4/settingsdialog.py:187
msgid "Cipher:"
msgstr "Cipher:"

#: ../../qt4/settingsdialog.py:193
msgid "Private Key:"
msgstr "Privát kulcs"

#: ../../qt4/settingsdialog.py:202
msgid "Key File"
msgstr ""

#: ../../qt4/settingsdialog.py:215 ../../qt4/settingsdialog.py:224
#: ../../qt4/settingsdialog.py:230
msgid "Password"
msgstr "Jelszó"

#: ../../qt4/settingsdialog.py:236
msgid "Save Password to Keyring"
msgstr "Jelszó mentése a kulcstartóba"

#: ../../qt4/settingsdialog.py:239
msgid "Cache Password for Cron (Security issue: root can read password)"
msgstr ""
"Jelszó mentése a Cron számára (biztonsági figyelmeztetés: a root látja a "
"jelszót)"

#: ../../qt4/settingsdialog.py:251
msgid "Advanced"
msgstr "Szakértői"

#: ../../qt4/settingsdialog.py:281 ../../qt4/settingsdialog.py:1646
msgid "Full snapshot path: "
msgstr ""

#: ../../qt4/settingsdialog.py:288
msgid "Schedule"
msgstr "Ütemterv"

#: ../../qt4/settingsdialog.py:298
msgid "Day:"
msgstr "Nap:"

#: ../../qt4/settingsdialog.py:309
msgid "Weekday:"
msgstr "Hét napja:"

#: ../../qt4/settingsdialog.py:320
msgid "Hour:"
msgstr "Óra:"

#: ../../qt4/settingsdialog.py:331
msgid "Hours:"
msgstr "Óra:"

#: ../../qt4/settingsdialog.py:340
msgid ""
"Run Back In Time repeatedly. This is useful if the computer is not running "
"regular."
msgstr ""

#: ../../qt4/settingsdialog.py:345
msgid "Every:"
msgstr ""

#: ../../qt4/settingsdialog.py:363
msgid ""
"Run Back In Time as soon as the drive is connected (only once every X "
"days).\n"
"You will be prompted for your sudo password."
msgstr ""
"Futtassa a Back In Time-ot, amint csatolod a meghajtót (X naponta egyszer).\n"
"Be fogja kérni a sudo jelszót."

#: ../../qt4/settingsdialog.py:376
msgid "Include"
msgstr "Hozzáadás"

#: ../../qt4/settingsdialog.py:382
msgid "Include files and folders"
msgstr "Felveendő fájlok és mappák"

#: ../../qt4/settingsdialog.py:401 ../../qt4/settingsdialog.py:456
msgid "Add file"
msgstr "Fájl hozzáadása"

#: ../../qt4/settingsdialog.py:405 ../../qt4/settingsdialog.py:460
msgid "Add folder"
msgstr "Mappa hozzáadása"

#: ../../qt4/settingsdialog.py:415
msgid "Exclude"
msgstr "Kihagyás"

#: ../../qt4/settingsdialog.py:418
msgid ""
"<b>Warning:</b> Wildcards ('foo*', '[fF]oo', 'fo?') will be ignored with "
"mode 'SSH encrypted'.\n"
"Only separate asterisk are allowed ('foo/*', 'foo/**/bar')"
msgstr ""
"<b>Figyelem:</b> a helyettesítő karaktreket ('foo*', '[fF]oo', 'fo?') az "
"'SSH encrypted' mód nem veszi figyelembe.\n"
"Csak az elválasztó csillag megengedett ('foo/*', 'foo/**/bar')"

#: ../../qt4/settingsdialog.py:426
msgid "Exclude patterns, files or folders"
msgstr "Kihagyandó minták, fájlok és mappák"

#: ../../qt4/settingsdialog.py:442
msgid "Highly recommended:"
msgstr "Erősen ajánlott:"

#: ../../qt4/settingsdialog.py:464
msgid "Add default"
msgstr ""

#: ../../qt4/settingsdialog.py:475
msgid "Exclude files bigger than: "
msgstr ""

#: ../../qt4/settingsdialog.py:476
#, python-format
msgid ""
"Exclude files bigger than value in %(prefix)s.\n"
"With 'Full rsync mode' disabled this will only affect new files\n"
"because for rsync this is a transfer option, not an exclude option.\n"
"So big files that has been backed up before will remain in snapshots\n"
"even if they had changed."
msgstr ""

#: ../../qt4/settingsdialog.py:494
msgid "Auto-remove"
msgstr "Automatikus eltávolítás"

#: ../../qt4/settingsdialog.py:501
msgid "Older than:"
msgstr "Régebbi, mint:"

#: ../../qt4/settingsdialog.py:516
msgid "If free space is less than:"
msgstr "Ha a szabad hely kevesebb, mint:"

#: ../../qt4/settingsdialog.py:529
msgid "If free inodes is less than:"
msgstr "Ha a szabad inode-ok száma kevesebb, mint"

#: ../../qt4/settingsdialog.py:552
msgid "Run in background on remote Host."
msgstr ""

#: ../../qt4/settingsdialog.py:555
msgid "Keep all snapshots for the last"
msgstr "Tartsd meg a pillanatképeket a legutóbbi:"

#: ../../qt4/settingsdialog.py:559 ../../qt4/settingsdialog.py:565
msgid "day(s)"
msgstr "napból"

#: ../../qt4/settingsdialog.py:561
msgid "Keep one snapshot per day for the last"
msgstr "Tarts meg napi egy pillanatképet a legutóbbi:"

#: ../../qt4/settingsdialog.py:567
msgid "Keep one snapshot per week for the last"
msgstr "Tarts meg heti egy pillanatképet a legutóbbi:"

#: ../../qt4/settingsdialog.py:571
msgid "weeks(s)"
msgstr "hétből"

#: ../../qt4/settingsdialog.py:573
msgid "Keep one snapshot per month for the last"
msgstr "Tarts meg havi egy pillanatképet a legutóbbi:"

#: ../../qt4/settingsdialog.py:577
msgid "month(s)"
msgstr "hónapból"

#: ../../qt4/settingsdialog.py:579
msgid "Keep one snapshot per year for all years"
msgstr "Tarts meg évi egy pillanatképet a legutóbbi:"

#: ../../qt4/settingsdialog.py:586
msgid "Don't remove named snapshots"
msgstr "Ne távolítsa el a névvel ellátott pillanatképeket"

#: ../../qt4/settingsdialog.py:598
msgid "Options"
msgstr "Beállítások"

#: ../../qt4/settingsdialog.py:603
msgid "Enable notifications"
msgstr "Értesítések engedélyezése"

#: ../../qt4/settingsdialog.py:606
msgid "Disable snapshots when on battery"
msgstr "Pillanatképek készítésének tiltása akkumulátorról működésnél"

#: ../../qt4/settingsdialog.py:609
msgid "Power status not available from system"
msgstr "A rendszer nem adja meg az energiaellátás állapotát"

#: ../../qt4/settingsdialog.py:612
msgid "Run only one snapshot at a time"
msgstr ""

#: ../../qt4/settingsdialog.py:613
msgid ""
"Other snapshots will be blocked until the current snapshot is done.\n"
"This is a global option. So it will effect all profiles for this user.\n"
"But you need to activate this for all other users, too."
msgstr ""

#: ../../qt4/settingsdialog.py:618
msgid "Backup replaced files on restore"
msgstr ""

#: ../../qt4/settingsdialog.py:619
#, python-format
msgid ""
"Newer versions of files will be renamed with trailing '%(suffix)s' before "
"restoring.\n"
"If you don't need them anymore you can remove them with '%(cmd)s'"
msgstr ""

#: ../../qt4/settingsdialog.py:628
msgid "Continue on errors (keep incomplete snapshots)"
msgstr "Folytatás hibák esetén (megtartja a nem teljes pillanatképeket)"

#: ../../qt4/settingsdialog.py:631
msgid "Use checksum to detect changes"
msgstr "Checksum használata hibák felfedezéséhez"

#: ../../qt4/settingsdialog.py:634
msgid "Full rsync mode. May be faster but:"
msgstr "Teljes rsync-mód. Lassabb lehet, de:"

#: ../../qt4/settingsdialog.py:635
msgid ""
"- snapshots are not read-only\n"
"- destination file-system must support all Linux attributes (dates, rights, "
"user, group ...)"
msgstr ""
"- a pillanatképek nem csak olvashatók\n"
"- a célfájlrendszernek támogatnia kell minden linux-attribútumot (dátumok, "
"jogok, felhasználó, csoport...)"

#: ../../qt4/settingsdialog.py:642
msgid "Take a new snapshot regardless of there were changes or not."
msgstr ""

#: ../../qt4/settingsdialog.py:645
msgid "Check for changes (don't take a new snapshot if nothing changed)"
msgstr ""
"Változások ellenőrzése (nem készít új pillanatképet, ha nincs változás)"

#: ../../qt4/settingsdialog.py:652
msgid "Log Level:"
msgstr "Naplózás szintje:"

#: ../../qt4/settingsdialog.py:657
msgid "None"
msgstr "Egyik sem"

#: ../../qt4/settingsdialog.py:659
msgid "Changes & Errors"
msgstr "Változások & Hibák"

#: ../../qt4/settingsdialog.py:675
msgid "Change these options only if you really know what you are doing !"
msgstr ""
"Ezeket a beállításokat csak akkor változtasd meg, ha biztosan tudod, mit "
"csinálsz!"

#: ../../qt4/settingsdialog.py:679
msgid "Run 'nice':"
msgstr ""

#: ../../qt4/settingsdialog.py:685 ../../qt4/settingsdialog.py:697
msgid "as cron job"
msgstr ""

#: ../../qt4/settingsdialog.py:688 ../../qt4/settingsdialog.py:703
#: ../../qt4/settingsdialog.py:717
msgid "on remote host"
msgstr ""

#: ../../qt4/settingsdialog.py:691
msgid "Run 'ionice':"
msgstr ""

#: ../../qt4/settingsdialog.py:700
msgid "when taking a manual snapshot"
msgstr ""

#: ../../qt4/settingsdialog.py:707
msgid "Run 'rsync' with 'nocache':"
msgstr ""

#: ../../qt4/settingsdialog.py:713
msgid "on local machine"
msgstr ""

#: ../../qt4/settingsdialog.py:720
msgid "Redirect stdout to /dev/null in cronjobs."
msgstr ""

#: ../../qt4/settingsdialog.py:726
msgid "Redirect stderr to /dev/null in cronjobs."
msgstr ""

#: ../../qt4/settingsdialog.py:735
msgid "Limit rsync bandwidth usage: "
msgstr "Az rsync sávszélesség-használatának korlátja: "

#: ../../qt4/settingsdialog.py:738
msgid " KB/sec"
msgstr " KB/sec"

#: ../../qt4/settingsdialog.py:773
msgid "Preserve ACL"
msgstr "ACL-ek megtartása"

#: ../../qt4/settingsdialog.py:786
msgid "Preserve extended attributes (xattr)"
msgstr "Kiterjesztett jogosultságok (xattr) megtartása"

#: ../../qt4/settingsdialog.py:804
msgid "Copy unsafe links (works only with absolute links)"
msgstr "Nem biztonságos linkek másolása (csak abszolút linkekkel működik)"

#: ../../qt4/settingsdialog.py:835
msgid "Paste additional options to rsync"
msgstr ""

#: ../../qt4/settingsdialog.py:838
msgid ""
"Options must be quoted e.g. --exclude-from=\"/path/to/my exclude file\"."
msgstr ""

#: ../../qt4/settingsdialog.py:848
msgid "Add prefix to SSH commands"
msgstr ""

#: ../../qt4/settingsdialog.py:851
#, python-format
msgid ""
"Prefix to run before every command on remote host.\n"
"Variables need to be escaped with \\$FOO.\n"
"This doesn't touch rsync. So to add a prefix\n"
"for rsync use \"%(cb_rsync_options)s\" with\n"
"%(rsync_options_value)s\n"
"\n"
"%(default)s: %(def_value)s"
msgstr ""

#: ../../qt4/settingsdialog.py:859 ../../qt4/settingsdialog.py:1697
msgid "default"
msgstr ""

#: ../../qt4/settingsdialog.py:876
msgid "Restore Config"
msgstr ""

#: ../../qt4/settingsdialog.py:892
msgid ""
"Full system backup can only create a snapshot to be restored to the same "
"physical disk(s) with the same disk partitioning as from the source; "
"restoring to new physical disks or the same disks with different "
"partitioning will yield a potentially broken and unusable system.\n"
"\n"
"Full system backup will override some settings that may have been "
"customized. Continue?"
msgstr ""

#: ../../qt4/settingsdialog.py:923
msgid "New profile"
msgstr "Új profil"

#: ../../qt4/settingsdialog.py:939
msgid "Rename profile"
msgstr "Profil átnevezése"

#: ../../qt4/settingsdialog.py:953
#, python-format
msgid "Are you sure you want to delete the profile \"%s\" ?"
msgstr "Biztosan törlöd a(z) \"%s\" profilt?"

#: ../../qt4/settingsdialog.py:1192
msgid ""
"Custom Hours can only be a comma seperate list of hours (e.g. 8,12,18,23) or "
"*/3 for periodic backups every 3 hours"
msgstr ""
"Az egyedi órák csak vesszővel elválasztott, listázott értékek lehetnek (pl. "
"8,12,18,23), vagy */3, ha háromóránként akarsz mentést készíteni."

#: ../../qt4/settingsdialog.py:1495
msgid "Exclude pattern"
msgstr "Kihagyási minta"

#: ../../qt4/settingsdialog.py:1508
msgid "Exclude file"
msgstr "Kihagyandó fájl"

#: ../../qt4/settingsdialog.py:1512
msgid "Exclude folder"
msgstr "Kihagyandó mappa"

#: ../../qt4/settingsdialog.py:1531
msgid "Include file"
msgstr "Felveendő fájl"

#: ../../qt4/settingsdialog.py:1539 ../../qt4/settingsdialog.py:1559
#, python-format
msgid ""
"\"%s\" is a symlink. The linked target will not be backed up until you "
"include it, too.\n"
"Would you like to include the symlinks target instead?"
msgstr ""

#: ../../qt4/settingsdialog.py:1551
msgid "Include folder"
msgstr "Felveendő mappa"

#: ../../qt4/settingsdialog.py:1578
msgid "Are you sure you want to change snapshots folder ?"
msgstr "Biztos, hogy szeretnéd megváltoztatni a pillanatképek mappáját?"

#: ../../qt4/settingsdialog.py:1694
msgid "enabled"
msgstr ""

#: ../../qt4/settingsdialog.py:1696
msgid "disabled"
msgstr ""

#: ../../qt4/settingsdialog.py:1722
msgid "Restore Settings"
msgstr ""

#: ../../qt4/settingsdialog.py:1739
msgid " and add your user to group 'fuse'"
msgstr ""

#: ../../qt4/settingsdialog.py:1743
#, python-format
msgid ""
"Please navigate to the snapshot from which you want to restore %(appName)s's "
"configuration. The path may look like: \n"
"%(samplePath)s\n"
"\n"
"If your snapshots are on a remote drive or if they are encrypted you need to "
"manually mount them first. If you use Mode SSH you also may need to set up "
"public key login to the remote host%(addFuse)s.\n"
"Take a look at 'man backintime'."
msgstr ""

#: ../../qt4/settingsdialog.py:1801 ../../qt4/settingsdialog.py:1864
msgid "No config found"
msgstr ""

#: ../../qt4/snapshotsdialog.py:49 ../../qt4/snapshotsdialog.py:173
msgid "Diff Options"
msgstr "Diff beállítások"

#: ../../qt4/snapshotsdialog.py:56
msgid "Command:"
msgstr "Parancs:"

#: ../../qt4/snapshotsdialog.py:60
msgid "Parameters:"
msgstr "Paraméterek:"

#: ../../qt4/snapshotsdialog.py:64
msgid "Use %1 and %2 for path parameters"
msgstr "Használja ezeket útvonal-paraméterként: %1 és %2"

#: ../../qt4/snapshotsdialog.py:106
msgid "List only different snapshots"
msgstr "Csak az eltérő pillanatképek listázása"

#: ../../qt4/snapshotsdialog.py:113
msgid "List only equal snapshots to: "
msgstr "Csak a következőkkel megegyező pillanatképek listázása: "

#: ../../qt4/snapshotsdialog.py:123
msgid "Deep check (more accurate, but slow)"
msgstr "Alapos ellenőrzés (pontosabb, de lassú)"

#: ../../qt4/snapshotsdialog.py:144
msgid "Delete"
msgstr "Töröl"

#: ../../qt4/snapshotsdialog.py:148
msgid "Select All"
msgstr "Minden kijelölése"

#: ../../qt4/snapshotsdialog.py:161
msgid "Diff"
msgstr "Különbség"

#: ../../qt4/snapshotsdialog.py:172
msgid "Go To"
msgstr "Ugrás ide:"

#: ../../qt4/snapshotsdialog.py:316
msgid "You can't compare a snapshot to itself"
msgstr "Egy pillanatkép nem hasonlítható össze önmagával"

#: ../../qt4/snapshotsdialog.py:323
#, python-format
msgid "Command not found: %s"
msgstr "A következő parancs nem található: %s"

#: ../../qt4/snapshotsdialog.py:344
#, python-format
msgid ""
"Do you really want to delete \"%(file)s\" in snapshot \"%(snapshot_id)s?\n"
msgstr ""
"Biztosan töröljem a \"%(file)s\" fájlt a \"%(snapshot_id)s pillanatképből?\n"

#: ../../qt4/snapshotsdialog.py:347
#, python-format
msgid "Do you really want to delete \"%(file)s\" in %(count)d snapshots?\n"
msgstr "Biztos, hogy törlöd a \"%(file)s\"-t a \"%(count)d-pillanatképből?\n"

#: ../../qt4/snapshotsdialog.py:349
msgid "WARNING: This can not be revoked!"
msgstr "Nem lehet visszavonni!"

#: ../../qt4/snapshotsdialog.py:364
#, python-format
msgid "Exclude \"%s\" from future snapshots?"
msgstr "Kizárjam a \"%s\"-t a jövőbeli pillanatképekből?"

#, python-format
#~ msgid "Compare with snapshot %s"
#~ msgstr "Összehasonlítás a következő pillanatképpel: %s"

#~ msgid "Remove old snapshots"
#~ msgstr "Régi pillanatképek eltávolítása"

#~ msgid ""
#~ "Update was successful!\n"
#~ "\n"
#~ "Back In Time will continue taking snapshots again as scheduled"
#~ msgstr ""
#~ "A frissítés sikeres volt.\n"
#~ "\n"
#~ "A Back in Time folytatja a pillanatképek készítését az időzítés szerint."

#~ msgid ""
#~ "Back In Time changed its backup format.\n"
#~ "\n"
#~ "Your old snapshots can be moved according to this new format. OK?"
#~ msgstr ""
#~ "A Back in Time  megváltoztatta a mentési formátumát.\n"
#~ "\n"
#~ "A régi pillanatképeket át tudjuk konvertálni ebbe az új formátumba. Rendben?"

#, python-format
#~ msgid ""
#~ "%s profiles found. \n"
#~ "\n"
#~ "The new backup format supports storage of different users and profiles on "
#~ "the same location. Do you want the same location for both profiles? \n"
#~ "\n"
#~ "(The program will still be able to discriminate between them)"
#~ msgstr ""
#~ "A(z) %s profilokat nem találom.\n"
#~ "\n"
#~ "Az új mentési formátum lehetővé teszi különböző felhasználók és profilok "
#~ "tárolását ugyanazon a helyen. Szeretnéd mindkét profilt ugyanazon a helyen "
#~ "tárolni?\n"
#~ "\n"
#~ "(A program ettől még meg fogja tudni különböztetni őket)"

#~ msgid ""
#~ "No profiles are found. Will have to update to profiles first, please restart "
#~ "Back In Time"
#~ msgstr ""
#~ "Nem találtam profilt. Először frissíteni kell a profilokat, először indítsd "
#~ "újra a Back In Time-ot"

#, python-format
#~ msgid ""
#~ "Former settings of profile %s are restored.\n"
#~ "Back In Time cannot continue taking new snapshots.\n"
#~ "\n"
#~ "You can manually move the snapshots, \n"
#~ "if you are done restart Back In Time to proceed"
#~ msgstr ""
#~ "A %s profil korábbi beállításai visszaállítva.\n"
#~ "A Back in Time nem tud újabb pillanatképeket készíteni.\n"
#~ "\n"
#~ "Kézzel tudod mozgatni a pillanatképeket,\n"
#~ "ha befejezted, a folytatáshoz indítsd újra a Back in Time-ot"

#~ msgid ""
#~ "Are you sure you do not want to move your old snapshots?\n"
#~ "\n"
#~ "\n"
#~ "If you do, you will not see these questions again next time, Back In Time "
#~ "will continue making snapshots again, but smart-remove cannot take your old "
#~ "snapshots into account any longer!\n"
#~ "\n"
#~ "If you do not, you will be asked again next time you start Back In Time."
#~ msgstr ""
#~ "Biztos, hogy nem akarod áthelyezni a pillanatképeket?\n"
#~ "\n"
#~ "\n"
#~ "Ha mégis,ezt a kérdést nem kapod meg újra, és a Back In Time újra készít "
#~ "majd pillanatképeket, de az okos eltávolítás nem veszi majd figyelembe a "
#~ "régi pillanatképeidet!\n"
#~ "\n"
#~ "Ha nem, a Back In Time következő indításakor újra megkapod ezt a kérdést."

#~ msgid "Back In Time will continue taking snapshots again as scheduled"
#~ msgstr "A Back in Time újra készít majd pillanatképeket az időzítés szerint."

#~ msgid ""
#~ "Back In Time still cannot continue taking new snapshots.\n"
#~ "Restart Back In Time to see the questions again"
#~ msgstr ""
#~ "A Back in Time nem tud újabb pillanatképeket készíteni.\n"
#~ "Indítsd újra a Back in Time -ot, hogy ismét láthasd a kérdéseket."

#~ msgid "Save config file ..."
#~ msgstr "Konfigurációs fájl mentése..."

#~ msgid "Save permission ..."
#~ msgstr "Hozzáférési jogosultságok mentése..."

#~ msgid "Try to keep min free space"
#~ msgstr "A minimálisan megadott szabad hely kialakítása"

#~ msgid "encfs not found. Please install e.g. 'apt-get install encfs'"
#~ msgstr "Nem találom az encfs-t. Telepítsd pl. így: 'apt-get install encfs'"

#, python-format
#~ msgid "Can't unmount encfs %s"
#~ msgstr "Nem tudom lecsatolni a(z) %s encfs-t"

#, python-format
#~ msgid "Can't unmount sshfs %s"
#~ msgstr "Nem tudom felcsatolni a(z) %s sshfs-t"

#~ msgid "sshfs not found. Please install e.g. 'apt-get install sshfs'"
#~ msgstr ""
#~ "Nem találom az sshfs-t. Telepítsd pl. az 'apt-get install sshfs' paranccsal."

#, python-format
#~ msgid ""
#~ "%(snapshots_left)s\n"
#~ "of profile %(profile_id)s are not moved to their new location\n"
#~ "Do you want to proceed?\n"
#~ "(Back In Time will be able to continue taking snapshots, however the "
#~ "remaining snapshots will not be considered for automatic removal)\n"
#~ "\n"
#~ "If not Back In Time will restore former settings for this profile, however "
#~ "cannot continue taking snapshots"
#~ msgstr ""
#~ "A(z) %(profile_id)s profil\n"
#~ "%(snapshots_left)s tartalma nem került át az új helyére.\n"
#~ "Folytassam?\n"
#~ "(A Back In Time továbbra is tud pillanatképeket készíteni, de a megmaradt "
#~ "pillanatképeket az automatikus eltávolítás figyelmen kívül fogja hagyni)\n"
#~ "Ha nem folytatom, a Back In Time visszaállítja ennek a profilnak a korábbi "
#~ "beállításait, de nem fog tudni pillanatképeket készíteni"

#, python-format
#~ msgid "Try to keep min %d%% free inodes"
#~ msgstr "Megpróbálok legalább %d%% szabad inode-ot tartani"
